apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -201,6 +203,12 @@ function WpcomBlockEditorNavSidebar() {
 						onClick={ dismissSidebar }
 					/>
 				</div>
+				<div className="wpcom-block-editor-nav-sidebar-nav-sidebar__site-title">
+					<h2>{ siteTitle }</h2>
+					<ExternalLink href={ SITE_HOME_URL }>
+						{ __( 'Visit site', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **124** times:
`translate( 'Visit site' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/wpcom-sidebar-title-home-url)